walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -19,5 +19,15 @@
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     throw new UnsupportedOperationException("TODO: Implement this method.");
+    //create an empty Collection<TimeRange> for available times for the requested meeting
+    //check if the duration of the event is more than a day or if the collection of attendees is empty
+    //if yes, return the empty Collection
+    //if no, begin looking for appropriate time slots
+    //Iterate through the day by checking each appropriate time duration ex. if meeting duration is 1hr 8-9, 8:30-9:30, ...
+    //for each time duration check if there is an event present and if anyone is already attending that meeting
+    //If the time is accepted, add it to the collection
```

Comment:
Ack, I just remembered an important detail. Based on the tests, they want all of the contiguous available times to be globbed together. So if it's a 1 hour meeting that could happen at 8:00, 8:30, or 9:00, they want that chunk of the day to be represented by a single `TimeRange` of 8:00 - 10:00. It seems possible to keep the described approach and then merge overlapping/adjacent time ranges at the end, but you should plan out how to do that, and if it becomes too messy, try a different approach for finding the time ranges in the first place.